tests/pages/dashboard/ComponentsList-cy.js

```diff
@@ -2,7 +2,8 @@ describe('ComponentsList [10b]', function () {
 
   context('No Components Found [10f]', function () {
 
-    it('shows error when components not found [10d]', function () {
+    // TODO: Turn into unit test
+    xit('shows error when components not found [10d]', function () {
```

Comment:
ahh i see. you can't force it to be empty. would it be possible for you to mock the response for tests where you need to mock it?